cumulusci/utils/git.py

```diff
@@ -81,6 +82,6 @@ def parse_repo_url(url: str) -> Tuple[str, str, str]:
 
     host = url_parts[-3]
     # Need to consider "https://api.github.com/repos/owner/repo/" pattern
-    if "http" in url_parts[0] and len(url_parts) > 6:
+    if "http" in url_parts[0] and len(url_parts) > 4 and not host.endswith(".com"):
```

Comment:
@mgrandhi This may work if hostname ends with .com, but if the user having hostname ends with different name then this may fail. Can we use regex instead of hard coding the value here?